### PR TITLE
Activities screen

### PR DESCRIPTION
**Pull request by GabrielMarquez0852**
- Setup Activities screen
- Included filtering and activity navigation

<p align="center">
  
  
</p>

### CODE REVIEW

**Review by jackzhen1996 — APPROVED** [review]
looks good!

File: App.js [hunk 1]
```diff
@@ -11,12 +11,14 @@ import Storytime from "./src/components/storytime.js";
 import MilkMilkMilk from "./src/components/milkMilkMilk";
 import Profile from "./src/components/profile.js";
 import Mindfulness from "./src/components/mindfulnessStack";
-import spriteChat from "./src/components/spriteChat";
+//import spriteChat from "./src/components/spriteChat";
```
Comment (latrujil913) [inline App.js:14]: Can you remove this comment

File: src/components/Activities.js [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+                                setNovice(false);
+                                setApp(false);
+                                setMaster(false);    
+    }
+
+    useEffect(() => {    
+        // This gets called after every render, by default  
+        // (the first one, and every one after that)    
+        //console.log('///////////UseEffect///// ' +  filter);  
+    })
```
Comment (latrujil913) [inline src/components/Activities.js:162]: Is this being used somehow? If not remove it

File: src/components/homeStack.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import Home from './home'
 import Calendar from './checkInStack'
 import Achievement from './Achievement'
-import Newsfeed from './Newsfeed'
+//import Newsfeed from './Newsfeed'
```
Comment (latrujil913) [inline src/components/homeStack.js:5]: Can you remove this comment

File: src/stylesheets/activitiesStyles.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        borderRadius: 10,
+    },
+    txtPressed: {
+        color: 'white',
+    },
+    txtDefult: {
+        color: '#A5A4A4'
+    },
+    headerView: {
+        //marginLeft: 26,
```
Comment (latrujil913) [inline src/stylesheets/activitiesStyles.js:44]: Remove commented out styles

**Review by latrujil913 — CHANGES_REQUESTED** [review]
Can you remove some commented out code. Also if you can add some comments at the top of your functions explaining what they are doing. (High level explination)

**Review by latrujil913 — APPROVED** [review]
Good stuff

**Pull request merged**